File: tournaments/higherVersion/higherVersion.py
def higherVersion(ver1, ver2):
    a = ver1.split(".")
    b = ver2.split(".")

    for i in range(0, len(a)):
        if int(a[i]) > int(b[i]):
            return True
        elif int(a[i]) < int(b[i]):
            return False

    return False

File: tournaments/higherVersion/higherVersion.js
function higherVersion(ver1, ver2) {
    const [a, b] = [ver1, ver2].map(ver => ver.split('.'));
    
    for (let i = 0; i in a; ++i) {
        if (+a[i] > +b[i]) {
            return true;
        } else if (+a[i] < +b[i]) {
            return false;
        }
    }
    
    return false;
}
